core/controllers/learner_dashboard.py

```diff
@@ -101,3 +102,35 @@ def get(self):
             'subscription_list': subscription_list
         })
         self.render_json(self.values)
+
+
+class LearnerDashboardFeedbackThreadHandler(base.BaseHandler):
+    """Gets all the messages in a thread."""
+
+    def get(self, exploration_id, thread_id):
+        """Handles GET requests."""
+        if self.user_id is None:
+            raise self.PageNotFoundException
+
+        messages_object = feedback_services.get_messages(
```

Comment:
messages_object --> messages (it's just a list, right? messages_object kind of suggests an instance of a Messages class)